bin/sdl.cc

```diff
@@ -220,12 +221,13 @@ int main(int argc, char** argv)
     ana.do_write_ntuple = result["write_ntuple"].as<int>();
 
     //_______________________________________________________________________________
-    // check if cpu library was loaded
+    // backends
     // 0 = cpu serial
     // 1 = cpu threads
     // 2 = cuda
     // 3 = hip
-    ana.do_run_cpu = SDL::getBackend() < 2;
+    ana.backend = result["backend"].as<int>();
+    ana.do_run_cpu = ana.backend < 2;
```

Comment:
From the last commits, I understand that the default (0) for the backend is CUDA. Isn't this line contradicting with that (at least conceptually)?

Reply:
That's a good point. It's not really affecting anything since `ana.do_run_cpu` only affects a printout, but it would be good to stick with some convention and have correct printouts.

Reply:
I got rid of this part in [aa92da0](https://github.com/SegmentLinking/TrackLooper/pull/377/commits/aa92da09ddf2dd7659e55e5bcfd87df9dd4cfdcc)